addons/sourcemod/scripting/szf/weapons.sp

```diff
@@ -139,25 +137,12 @@ void Weapons_ReplaceEntityModel(int ent, int index)
 	}
 }
 
-// enum structs are truly epic
-void Weapons_GetOffsets(eWeapon wep, float origin[3], float angles[3])
-{
-	origin[0] = wep.flOffsetOrigin[0];
-	origin[1] = wep.flOffsetOrigin[1];
-	origin[2] = wep.flOffsetOrigin[2];
-	
-	angles[0] = wep.flOffsetAngles[0];
-	angles[1] = wep.flOffsetAngles[1];
-	angles[2] = wep.flOffsetAngles[2];
-}
-
 // -----------------------------------------------------------
 public bool Weapons_OnPickup_Health(int client)
 {
 	if (GetClientHealth(client) < SDK_GetMaxHealth(client))
 	{
 		SpawnPickup(client, "item_healthkit_full");
-		EmitSoundToClient(client, "items/smallmedkit1.wav");
```

Comment:
need to readd sound to config

Reply:
this(or the similar) sound already gets played when player picks up the medkit entity